### DISCUSSION TITLE
Browser bookmark

### DISCUSSION THREAD

**Discussion by kaymyst** [isontheline/pro.webssh.net#1136] [Ideas]
it would be great to have a way to save URLs on the browser tool so that you can come back to it quickly on a phone.
I use the port forward mainly and having to type a link in the address bar of the browser is a bit cumbersome
thanks
great tool BTW

**Comment by kaymyst** [comment]
TBF I worked out how to do VPN over Ssh and got a bookmark saved on my phone desktop now so it does the job

**Reply by isontheline** [reply]
Hello @kaymyst 👋

Thank you so much for reaching me out 🙏
I have opened weeks ago a similar issue : https://github.com/isontheline/pro.webssh.net/issues/1056
I agree with you that a bookmark manager would be a nice thing to have
